### ISSUE TITLE
[Orbit] [Magellan] etc. adding data-options does not work

### ISSUE THREAD

**Issue by phifa** [zurb/foundation-sites#7258]
Adding data-options to any js element does not seem to work.

**Comment by zurbchris** [comment]
It works on my machine, what formatting are you using?

**Comment by phifa** [comment]
For example adding this does nothing:  
```
<ul class="dropdown vertical large-horizontal menu mainmenu" id="mainmenu" data-magellan data-threshold="-1">
```
instead if I use it programmaticall it works: 
```
var $mainmenu = new Foundation.Magellan($('#mainmenu'), {
    threshold: -1
});
```

**Comment by zurbchris** [comment]
Fixed with cd6f54f
